cmd/tiller/release_server.go

```diff
@@ -302,7 +307,7 @@ func (s *releaseServer) uniqName(start string, reuse bool) (string, error) {
 			return "", fmt.Errorf("release name %q exceeds max length of %d", start, releaseNameMaxLen)
 		}
 
-		if rel, err := s.env.Releases.Get(start); err == driver.ErrReleaseNotFound {
+		if rel, err := s.env.Releases.Get(start, 0); err == driver.ErrReleaseNotFound {
```

Comment:
I'm getting a panic in the in-memory storage option from this query (see below). Should be 1 instead of 0 ?
```
2016-08-24 18:28:11.520108 I | Getting release "hello" (v0) from storage
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x10 pc=0x5cb9]

goroutine 25 [running]:
panic(0x1670160, 0xc8200100e0)
       	/usr/local/go/src/runtime/panic.go:464 +0x3e6
main.(*releaseServer).uniqName(0xc8204fe248, 0xc82033d1e0, 0x5, 0x1a7200, 0x0, 0x0, 0x0, 0x0)
       	/Users/michellenoorali/gocode/src/k8s.io/helm/cmd/tiller/release_server.go:312 +0x329
main.(*releaseServer).prepareRelease(0xc8204fe248, 0xc8204f2fa0, 0x40, 0x0, 0x0)
       	/Users/michellenoorali/gocode/src/k8s.io/helm/cmd/tiller/release_server.go:372 +0xa6
main.(*releaseServer).InstallRelease(0xc8204fe248, 0x2c9c050, 0xc82047fcb0, 0xc8204f2fa0, 0xc82003da80, 0x0, 0x0)
       	/Users/michellenoorali/gocode/src/k8s.io/helm/cmd/tiller/release_server.go:353 +0x4a
k8s.io/helm/pkg/proto/hapi/services._ReleaseService_InstallRelease_Handler(0x183a8e0, 0xc8204fe248, 0x2c9c050, 0xc82047fcb0, 0xc8200125c0, 0x0, 0x0, 0x0, 0x0)
```